invenio_rdm_records/services/review/service.py

```diff
@@ -146,8 +149,11 @@ def submit(self, identity, id_, data=None, revision_id=None, uow=None):
         if draft.parent.review is None:
             raise ReviewNotFoundError()
 
+
         # All other preconditions can be checked by the action itself which can
         # raise appropriate exceptions.
-        request = current_requests_service.execute_action(
+        request_item = current_requests_service.execute_action(
             identity, draft.parent.review.id, 'submit', data=data, uow=uow)
-        return request
+        draft.parent.review = request_item._request
```

Comment:
Ditto to why this assignment is needed?

Reply:
Add comment as above for this assignment